app/helpers/trainee_helper.rb

```diff
@@ -18,12 +18,18 @@ def view_trainee(trainee)
 
   def trainees_page_title(trainees)
     total_pages = trainees.total_pages
-    return "trainees.index" if total_pages <= 1
+    trainees_count_text = @filtered_trainees.count.to_s + " record".pluralize(@filtered_trainees.count)
```

Comment:
We should calculate and pass in the count as an argument to this helper function, rather than reply on the instance variable `@filtered_trainees` having been created in the controller.

Something like:

```
def trainees_page_title(trainees, total_trainees_count)
  ...
end
```

Reply:
That should appease the linter too!